(@W-20241699) Add new policy parameters (#309)
mulesoft/docs-anypoint-cli: modules/ROOT/pages/_partials/api-mgr.adoc

[%header%autowidth.spread,cols="a,a,a"]
|===
|Flag |Description| Example

> | `--allowDuplicated` | Indicates whether the Policy can be applied multiple times on the API Instance. | `--allowDuplicated`

Reviewer: ```suggestion
| `--allowDuplicated` | Indicates whether the policy can be applied multiple times on the API Instance. | `--allowDuplicated`
```

| `--groupId`
| Mule 4 policy group ID +
This value defaults to the MuleSoft group ID. | `--groupId 1fec0a49-1551-4199-bfcc-cf0352d0f29d`

| `--label`

> | An optional label to identify this policy instance | `--label user authentication policy`

Reviewer: ```suggestion
| Optional label to identify the policy instance | `--label user authentication policy`
```

[%header%autowidth.spread,cols="a,a,a"]
|===
|Flag |Description| Example

> | `--allowDuplicated` | Indicates whether the Policy can be applied multiple times on the API Instance. | `--allowDuplicated`

Reviewer: ```suggestion
| `--allowDuplicated` | Indicates whether the policy can be applied multiple times on the API Instance. | `--allowDuplicated`
```

| `-c`, `--config`
| Pass the configuration data as a JSON string |
`--config '{\"username\":\"user\",\"password\":\"test\"}'`

| `--configFile`
| Pass the configuration data as a file |
`--configFile ./config.json`

| `--label`

> | An optional label to identify this policy instance | `--label user authentication policy`

Reviewer: ```suggestion
| Optional label to identify the policy instance | `--label user authentication policy`
```
